### ISSUE TITLE
alter_table does not work

### ISSUE THREAD

**Issue by joshuanapoli** [greglu/hbase-stargate#7]
The gem defines an alter_table operation, but it appears to be unfinished. It calls an undefined method 'construct_xml_stream'.

**Comment by greglu** [comment]
You're absolutely right and I believe the method is a remnant of the previous hbase-ruby library I based my earlier work on. I'll be implementing this method soon but for now, I'm having it throw a NotImplementedError. Stay tuned!
